ReactSkia/views/common/RSkImageCacheManager.cpp

```diff
@@ -171,5 +171,38 @@ bool RSkImageCacheManager::imageDataInsertInCache(const char* path,decodedimageC
   }
 }
 
+bool RSkImageCacheManager::clearMemory() {
+  std::scoped_lock lock(imageCacheLock);
+  size_t usageArr[2]={0,0};
+  ImageCacheMap::iterator it=imageCache_.begin();
+  while( it != imageCache_.end()) {
+    if((it->second.imageData)->unique()) {
+      it=imageCache_.erase(it);
+    } else {
+      ++it;
+    }
+  }
+  imageCache_.clear();
+#ifdef RNS_SHELL_HAS_GPU_SUPPORT
+  RnsShell::WindowContext::grTransactionBegin();
+  GrDirectContext* gpuContext = RSkSurfaceWindow::getDirectContext();
+  if(gpuContext)
+   gpuContext->purgeUnlockedResources(false);
+  RnsShell::WindowContext::grTransactionEnd();
+#endif
+#ifdef RNS_IMAGE_CACHE_USAGE_DEBUG
+    printCacheUsage(); // for print memory size for CPU and GPU cache
+#endif //RNS_IMAGECACHING_DEBUG
+  SkGraphics::PurgeResourceCache(); // purge for CPU memory cache
```

Comment:
does this work on non-debug mode ? as per document this api is for debugging purpose

Reply:
In SkGraphics.h file there is no debugging micro found to say this is debugging micro/API. for other APIs in the .h file, they used the same API to purge the cache. so we can use this API